recipes/heliano/build.sh

```
@@ -0,0 +1,41 @@
+sed -i.bak "s|_SPLIT_JOINT_|${SPLIT}|" ${PACKAGE_HOME}/heliano.py
+sed -i.bak "s|_SORTPRO_|${SORT}|" ${PACKAGE_HOME}/heliano.py
+
+# set permissions to files
+chmod +x ${PACKAGE_HOME}/heliano.py
+chmod +x ${PACKAGE_HOME}/heliano_cons.py
+
+# test for conda
+df -h
+
+# put files in the executable bin
+cd ${PREFIX}/bin
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)